jwst/datamodels/schemas/keyword_pband.schema.yaml

```diff
@@ -9,5 +9,6 @@ properties:
           p_band:
             title: MRS wavelength band
             type: string
-            pattern: ^((ANY|SHORT|MEDIUM|LONG|N/A)\s*\|\s*)+$
+            pattern: ^((ANY|SHORT|MEDIUM|LONG|SHORT-MEDIUM|SHORT-LONG|\
+	    MEDIUM-SHORT|MEDIUM-LONG|LONG-SHORT|LONG-MEDIUM|N/A)\s*\|\s*)+$
```

Comment:
Same indentation problem here. Must be indented, like line 17 in keyword_readpatt.schema.yaml (below).